delivery_price_rule_untaxed/tests/test_delivery_price_rule_untaxed.py

```diff
@@ -53,5 +53,5 @@ def test_untaxed(self):
 
         self.assertEqual(len(line), 1,
                          "Delivery cost is not Added")
-        self.assertEqual(line.price_subtotal, 2 * 300,
+        self.assertEqual(line.price_unit, 2 * 300,
```

Comment:
Why this change ?

Reply:
Honestly i only opened pr and squashed commits, i'll revert this change if needed